src/components/home/Features.tsx

```diff
@@ -14,30 +15,35 @@ type FeatureProps = {
   }
 }
 
-const Feature = ({feature}: FeatureProps) => {
+const Feature = ({feature}: FeatureProps): JSX.Element => {
   return (
-    <div key={feature.id} className="text-tailCall-dark-500 sm:max-w-6xl sm:m-10 lg:mx-auto lg:px-16 my-0 lg:my-20">
-      <div className="flex items-center -ml-2 sm:space-x-2 sm:-ml-5 lg:-ml-10 mb-6">
+    <div
+      key={feature.id}
+      className="text-tailCall-dark-500 sm:max-w-6xl sm:m-SPACE_10 lg:mx-auto lg:px-SPACE_016 my-0 lg:my-SPACE_20"
+    >
+      <div className="flex items-center -ml-SPACE_02 sm:space-x-SPACE_02 sm:-ml-SPACE_05 lg:-ml-SPACE_10 mb-SPACE_06">
         <GreaterThanUnderscoreIcon className="h-5 sm:h-7" />
-        <h5 className="text-title-semi-large sm:text-display-tiny lg:text-display-small mb-0">{feature.title}</h5>
+        <Heading as="h5" className="text-title-semi-large sm:text-display-tiny lg:text-display-small mb-0">
+          {feature.title}
+        </Heading>
       </div>
 
       <div
-        className="flex flex-col sm:flex-row p-4 sm:p-10  shadow-xl rounded-[20px] relative  sm:ml-0"
+        className="flex flex-col sm:flex-row p-SPACE_04 sm:p-SPACE_16  shadow-xl rounded-[20px] relative  sm:ml-0"
         style={{
```

Comment:
style is used with classname

Reply:
This is intentional we are using custom tailwind spacing tokens for paddings, margins, spacing & positions.